vdr/types/common.go

```diff
@@ -52,6 +52,8 @@ type DocumentMetadata struct {
 	Updated *time.Time `json:"updated,omitempty"`
 	// Hash of DID document bytes. Is equal to payloadHash in network layer.
 	Hash hash.SHA256Hash `json:"hash"`
+	// Transactions points to the transaction(s) that created this DID Document. If multiple transactions are listed, the DID Document is conflicted
```

Comment:
"that created this document" is only true for the initial transaction?

Isn't it more like "points to the previous versions of the DID document, or the current version in case of a new DID document"?

Reply:
"that created the current version of this DID Document."